documentation/smg/06_deployment_process.md

```diff
@@ -0,0 +1,79 @@
+# Distributing Horace to users
+
+Current Horace distribution process uses two channels, which allows us frequently modify the code,
+satisfy the requests of different groups of users and rapidly react to the bugs, encountered by users
+running their experiments in ISIS. The distribution is deployed  
+from live Horace/Herbert code tree after a developer added the requested features and 
+run existing unit/system tests ensuring no trivial bugs are 
+introduced at the development stage.
+
+The channels are the *compressed zip archive* downloaded from the web and intended for external users
+and *updates to git **production** branch* located on parallel file system and intended for the users,
+who are doing the experiments in ISIS.
+
+
+## Zip archive channel.
+
+The process of installing zip-archive from the user perspective is described in details at 
+[Horace Distribution page.](http://horace.isis.rl.ac.uk/Download_and_setup)
+
+To synchronize zip-archive distribution channel with the production branch distribution channel, 
+developer should merge all necessary changes into the production branch of the git archive
+and do the following work from this branch. (Horace and Herbert production branches together)
+
+After ensuring that all tests are passing on the production branches, the developer should leave 
+the horace/herbert code tree and move his Matlab working folder to a location, where the Horace 
+distributive should be located. After that, he runs the `make_horace_deployment_kit` script, 
```

Comment:
"and move his Matlab working folder to a location, where the Horace distributive should be located" I don't understand what this requires me to move/do.